server/test/services/program/ProgramDefinitionTest.java

```diff
@@ -1416,8 +1373,6 @@ public void serializeThenDeserialize_allFieldsPresent() throws JsonProcessingExc
             .setCreateTime(Instant.now())
             .setLastModifiedTime(Instant.now())
             .build();
-    applicationStatusesRepository.createOrUpdateStatusDefinitions(
```

Comment:
question (non-blocking): Why remove this?

Reply:
Statuses are no longer part of Program Definitions. We should not be testing this here.